index.bs

```diff
@@ -2626,6 +2560,24 @@ needs.
         interest of minimizing disruption to the user interaction flow).
 </div>
 
+### Client Supported Capability Enumeration (enum <dfn enum>SupportedCapability</dfn>) ### {#enum-supportedcapability}
+
+<xmp class="idl">
+    enum SupportedCapability {
+        "clientPin",
+        "residentKeys"
+    };
+</xmp>
+
+<div dfn-type="enum-value" dfn-for="SupportedCapability">
+    Each enum value indicates a supported capability of the [=client platform=] for use by [=[WRPS]=].
+    :   <dfn>clientPin</dfn>
+    ::  This attribute indicates whether the [=client platform=] supports [=user verification=] via client PIN.
+
+    :   <dfn>residentKeys</dfn>
+    ::  This attribute indicates whether the [=client platform=] supports creating [=resident credentials=].
```

Comment:
This is also about whether the client supports resident credential selection (for external authenticators without their own display), right? Or should that be a separate capability?

Reply:
@leshi, what do you mean by "account selection" ?

Reply:
(5), key management, isn't standardised in WebAuthn (yet?), so I'd say we can't make that requirement at this time.